src/styles/blocks/catalog.scss

```
@@ -11,6 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: 300ms ease;
```

By default transition is used for all. Needs to set only what changes
```suggestion
    transition: transform 0.3s ease, color  0.3s ease;
```